### PR TITLE
List UI

### PR DESCRIPTION
**Pull request by EveThan**
- Added logo on login page (might have to adjust its position a little coz it's too high up)
- Added code for implementing "swipe to delete" for own mood list
- Haven't fully implemented the delete action yet (click on the delete button won't do anything)

Sorry if I make any stupid mistakes! I have tried to make time for this😿

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -39,6 +39,7 @@ task javadoc(type: Javadoc) {
 }
 
 dependencies {
+    implementation 'com.baoyz.swipemenulistview:library:1.3.0'
```
Comment (jenyangk) [inline app/build.gradle:42]: What's this for?

**Review by jenyangk — COMMENTED** [review]
You might want to let others know that you're using a special ListView and also if you could remove the `google.services.json.zip` from this PR. @liushuyu might be able to help you if you're not sure how.

**Review by liushuyu — CHANGES_REQUESTED** [review]
- You introduced a token file, please remove it immediately. You need to erase this file from the entire history where it first got introduced
- When introducing a new library, take a look if it's compatible with `androidx` and how well it got updated in the last few months

File: app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -8,15 +8,14 @@
...
-        app:layout_constraintEnd_toEndOf="parent"
-        android:id="@+id/btn_sign_in"
-        android:layout_marginBottom="48dp"
-        app:layout_constraintBottom_toBottomOf="parent"
+        android:onClick="showLogin"
         android:text="@string/login_sign_in"
-        android:onClick="showLogin" />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:layout_editor_absoluteY="635dp" />
```
Comment (liushuyu) [inline app/src/main/res/layout/activity_login.xml:18]: This element isn't vertically bounded

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java [hunk 3]
```diff
@@ -32,6 +48,45 @@ public View onCreateView(@NonNull LayoutInflater inflater,
...
+                menu.addMenuItem(deleteItem);
+            }
+        };
+
+        moodListView.setMenuCreator(creator);
+
+        moodListView.setOnMenuItemClickListener(new SwipeMenuListView.OnMenuItemClickListener() {
+            @Override
+            public boolean onMenuItemClick(int position, SwipeMenu menu, int index) {
+                switch (index) {
```
Comment (liushuyu) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java:80]: Potentially useless `switch` statement?

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java [hunk 4]
```diff
@@ -32,6 +48,45 @@ public View onCreateView(@NonNull LayoutInflater inflater,
...
+        moodListView = mView.findViewById(R.id.mood_list);
+        MoodEventCustomList adapter = new MoodEventCustomList(mActivity, moodEventList);
+        moodListView.setAdapter(adapter);
+
+        SwipeMenuCreator creator = new SwipeMenuCreator() {
+            @Override
+            public void create(SwipeMenu menu) {
+                // create "delete" item
+                @SuppressLint("RestrictedApi") SwipeMenuItem deleteItem = new SwipeMenuItem(
+                        getApplicationContext());
```
Comment (liushuyu) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java:62]: Why is `com.firebase.ui.auth.AuthUI.getApplicationContext` used there?

**Review by liushuyu — CHANGES_REQUESTED** [review]
- Also, your branch isn't up-to-date with the master branch, please merge or rebase your changes against the `master` branch

**Pull request closed without merging**